pushtemplates/src/main/java/com/clevertap/pushtemplates/PTNotificationIntentService.java

```diff
@@ -71,6 +71,7 @@ private void handleActionButtonClick(Bundle extras) {
                 }
 
             }
+            Utils.raiseNotificationClicked(getApplicationContext(), extras);
```

Comment:
@vagnihotri We don't need this here, `PTNotificationIntentService` will eventually open the app with the DL info and notification clicked will be raised by the `ActivityLifecycleCallbacks` of the app